lib/ruby_lsp/ruby_lsp_rails/support/rails_document_client.rb

```diff
@@ -0,0 +1,122 @@
+# typed: strict
+# frozen_string_literal: true
+
+require "net/http"
+
+module RubyLsp
+  module Rails
+    module Support
+      class RailsDocumentClient
+        RAILS_DOC_HOST = "https://api.rubyonrails.org"
+        SUPPORTED_RAILS_DOC_NAMESPACES = T.let(
+          Regexp.union(
+            /ActionDispatch/,
+            /ActionController/,
+            /AbstractController/,
+            /ActiveRecord/,
+            /ActiveModel/,
+            /ActiveStorage/,
+            /ActionText/,
+            /ActiveJob/,
+          ).freeze,
+          Regexp,
+        )
+
+        RAILTIES_VERSION = T.let(
+          [*::Gem::Specification.default_stubs, *::Gem::Specification.stubs].find do |s|
+            s.name == "railties"
+          end&.version&.to_s,
+          T.nilable(String),
+        )
+
+        class << self
+          extend T::Sig
+          sig do
+            params(name: String).returns(T::Array[String])
+          end
```

Comment:
```suggestion
          extend T::Sig
          
          sig { params(name: String).returns(T::Array[String]) }
```

Reply:
I think this is such a common thing that we should have a cop for it? I can create an issue if there isn't one already.

Reply:
I don't think there's an issue for it. Not sure how common it is, but definitely good to maintain standards.

Reply:
https://github.com/Shopify/rubocop-sorbet/issues/174